### PR TITLE
Add browser builds

### LINKED ISSUE #279: Builds for browsers
We need builds for browsers with transpiring to different dialects, optimization and minification. @nechaido can you please research and implement this? I hope @aqrln can help you.

### PR DESCRIPTION
**Pull request by nechaido**
Fixes #279.

### CODE REVIEW

**Comment by nechaido** [comment]
@tshemsedinov we will need to have separate `metarhia.browser.js` then, I am ok with both variants.

**Comment by tshemsedinov** [comment]
@nechaido agreed

**Comment by nechaido** [comment]
@tshemsedinov fixed.

**Comment by tshemsedinov** [comment]
ping @aqrln

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  entry: ['babel-polyfill', './dist/metasync.js'],
+  output: {
+    path: path.join(__dirname, distDir),
+    filename: 'metasync.min.js',
+    sourceMapFilename: 'metasync.min.js.map',
+    library: 'metasync',
+  },
+  devtool: 'source-map',
+  plugins: [
+    new UglifyJsPlugin({
```
Comment (aqrln) [inline webpack.config.js]: Heads up: there's no `UglifyJsPlugin` in webpack 4 (webpack 3 is used in the PR). It's now built into webpack core and has its own field in the config, which one would rarely use directly as simply `mode: 'production'` enables both this and many other optimizations.

File: metasync.browser.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use strict';
```
Comment (aqrln) [inline metasync.browser.js]: This shouldn't be added to the repo, but generated with Babel on `prepublish` instead.

**Comment by tshemsedinov** [comment]
@nechaido rebese please

**Comment by nechaido** [comment]
Landed in 56dd8aa.

**Pull request closed without merging**